tests/UI/campaigns/productV2/sanity/03_CRUDProductWithCombinations.js

```diff
@@ -0,0 +1,278 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+
+// Import utils
+const helper = require('@utils/helpers');
+const testContext = require('@utils/testContext');
+
+// Import login steps
+const loginCommon = require('@commonTests/BO/loginBO');
+const {enableNewProductPageTest, disableNewProductPageTest} = require('@commonTests/BO/advancedParameters/newFeatures');
+
+// Import BO pages
+const dashboardPage = require('@pages/BO/dashboard');
+const productsPage = require('@pages/BO/catalog/productsV2');
+const createProductsPage = require('@pages/BO/catalog/productsV2/add');
+
+// Import FO pages
+const foProductPage = require('@pages/FO/product');
+
+// Import faker data
+const ProductFaker = require('@data/faker/product');
+
+const baseContext = 'productV2_sanity_CRUDProductWithCombinations';
+
+let browserContext;
+let page;
+
+// Data to create product with combinations
+const newProductData = new ProductFaker({
+  type: 'combinations',
+  quantity: 50,
+  minimumQuantity: 1,
+  status: true,
+});
+// Data to edit product with combinations
+const editProductData = new ProductFaker({
+  type: 'combinations',
+  quantity: 100,
+  minimumQuantity: 1,
+  status: true,
+  combinations: {
```

Comment:
```suggestion
  attributes: {
```

These are the attributes selected based on which the combinations will be generated. So I think renaming this would reflect more what the test is doing. In the same spirit `setProductCombinations` could be renamed `selectAttributes` and `addCombination` -> `selectAttribute`

Reply:
Thank you @jolelievre for the review,
For changing combinations by attributes I should change the object name in the whole project, so I prefer to change it ASAP in another PR.
Done for renaming methods :+1: